### PR TITLE
Add `lr_scheduler` helper functions

### PR DESCRIPTION
**Pull request by lightaime**
Add a helper to get any learning rate scheduler implemented in pyg and pytorch from its name or type.

### CODE REVIEW

**Comment by lightaime** [comment]
Drop this in favor of https://github.com/pyg-team/pytorch_geometric/pull/5942.
